src/components/draft

```diff
@@ -1,16 +1,147 @@
-class Cat extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {sleeping: true};
-  }
-  render() {
-    return (
-      <div>
-        <p>{`The cat is ${this.state.sleeping ? "sleeping" : "eating"}`}</p>
-        <button onClick={() => this.setState({ sleeping: false })}>
-          Wake up!
-        </button>
-      </div>
-    );
-  }
-}
\ No newline at end of file
+import React, { useState } from 'react';
```

Comment:
- [ ] I can see that you're using this file as a draft for your code. I liked the idea 🤩 Great job overall 👏 However, I highly recommend that you delete the file for a more clean code base 😄 🙏 You might save the file outside the app dir!

Reply:
@erezfree29 Please let's work on the issue mentioned by the previous reviewer.